static/css/style.css

```diff
@@ -1887,37 +1887,60 @@ input.input-has-error {
   margin: auto 3px;
 }
 
-.column-blocked {
+.relay-stat {
   position: relative;
 }
 
 .nested-grid {
   cursor: default;
 }
 
-.blocked-description.show-message {
-  visibility: visible;
-  transition: all 0.3s ease;
-}
-
-.blocked-description {
+.stat-description {
   position: absolute;
   visibility: hidden;
-  bottom: -66px;
-  left: 20px;
+  transform: translateY(O);
+  bottom: -70px;
+  right: 19px;
   font-size: 13px;
-  line-height: 1.5;
+  line-height: 1.25;
   width: 260px;
   background-color: var(--grey10);
   border: 1px solid var(--grey30);
   box-shadow: var(--dropShadowLight);
   padding: 8px 16px;
   border-radius: 4px;
   z-index: 2;
+  opacity: 0;
+  transition: all 0.2s ease !important;
+}
+
+.stat-description.show-message {
+  transform: translateY(3px);
+  visibility: visible;
   opacity: 1;
   transition: all 0.3s ease;
 }
 
+.stat-description.blocked-description {
```

Comment:
Would this need to apply to `.stat-description.forwarding-description` as well?

Reply:
Nope, just `.blocked-description`.